Public/Src/IDE/VsCode/BuildXL.IDE.VsCode.dsc

```diff
@@ -55,6 +56,10 @@ namespace LanguageService.Server {
         let json = IDE.VersionUtilities.updateVersion(version, f`client/package.json`);
         let readme = IDE.VersionUtilities.updateVersion(Branding.version, f`client/README.md`);
 
+        const extensionSourceFolder = Context.getNewOutputDirectory(`VsixTemp`);
+        Npm.installFromPackageJson(d`client`, d`${extensionSourceFolder}/node_modules`);

```

Comment:
To establish a dependency between pips, these functions should return some output artifacts and then you should use those artifacts down below to specify vsix deployment.  The way it is now there is no dependency between the deployment pip and these `npm install`/`npm run compile` pips; consequently, when you run a build these `npm` pips are not even executed